### ISSUE TITLE
Missing icon: Gentoo 

### ISSUE THREAD

**Issue by chino23** [ryanoasis/nerd-fonts#250]
Unicode f30d as shown on the cheat-sheet, the gentoo logo isn't outputted.

Instead, Ubuntu icon is outputted.

Occurs on Dejavu Sans Mono as well as MesloLGM.

**Comment by kajzersoze** [comment]
MesloLGM prints gentoo icon fine here.
Didn't try Dejavu Sans Mono.

**Comment by chino23** [comment]
that's most weird. why would it print sth else for me?

**Comment by kajzersoze** [comment]
Have no clue, I'm using latest v2.0.0 but it was printing gentoo icon with previous versions as well. How are you printing that icon ?
`echo "\uf30d"`
works in my case.

**Comment by chino23** [comment]
I use the same command. check this out
![gentoo](https://user-images.githubusercontent.com/7042593/37918110-6ae845cc-3120-11e8-995e-4b9db6997a9b.png)

**Comment by kajzersoze** [comment]
Yeah well, interesting, unfortunately I can't help you with that problem, try to reinstall the fonts maybe, dunno...
Anyway, I just tried DejaVuSansMono Nerd Font and that one works too.


![20180326-182352](https://user-images.githubusercontent.com/11412353/37919150-67e0dbb6-3123-11e8-86dd-b7db74e4aee8.png)

**Comment by chino23** [comment]
ok thanks, i'll try reinstalling everything
